dmapiclient/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = '17.2.0'
+__version__ = '17.4.0'
```

Comment:
Did we miss `17.3.0`?

Reply:
I bumped it twice, once when adding `_patch` and once when adding `mark_direct_award_still_assessing`. I can change it to one bump though.